modules/porous_flow/include/materials/PorousFlowMaterial.h

```diff
@@ -73,4 +79,6 @@ class PorousFlowMaterial : public Material
   const VariableName _saturation_variable_name;
   const VariableName _temperature_variable_name;
   const VariableName _mass_fraction_variable_name;
+
+  std::vector<unsigned int> _qpmap{{0, 1, 2, 3}};
```

Comment:
```suggestion
```

Reply:
Oops - forgot to delete this after using it for some debugging